test/LP/LP.jl

```diff
@@ -96,30 +135,32 @@ const LP_TEST_FILE = "test.lp"
         end
 
         @testset "Too long duplicate names after sanitization" begin
-            model = LP.Model()
+            max_length = 255
```

Comment:
Instead of 255, you could use a much shorter `max_length` for the test (like 5).